src/api/playlist.ts

```
@@ -31,6 +31,11 @@ export const postPlaylist = (body: PostPlaylistRequest): Promise<PostPlaylistRes
     return postFetch<PostPlaylistResponse, string>("/playlist", body).then(res => res ?? false)
 };
 
+
+export const postPlaylistNowPlaying = (playlistCode: string, videoCode : string): Promise<boolean | {}> => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)